### PR TITLE
Add user and group params

### PR DESCRIPTION
**Pull request by akondas**

### CODE REVIEW

File: handlers/main.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # handlers file for supervisor
 ---
 - name: restart supervisor
+  become: yes
```
Comment (tersmitten) [inline handlers/main.yml]: `become` does not work in Ansible < 1.9. Can't this be specified at playbook level?

Comment (akondas) [inline handlers/main.yml]: Yes it can. I remove this.

**Comment by tersmitten** [comment]
Thanks for the contribution. I would love to merge it, but can you make the `README` up-2-date?

**Comment by akondas** [comment]
now it should be complete

**Pull request merged**